networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/DataColumnSidecarsByRootMessageHandler.java

```diff
@@ -254,6 +255,14 @@ private SafeFuture<Optional<DataColumnSidecar>> retrieveDataColumnSidecar(
     return dataColumnSidecarCustodySupplier
         .get()
         .getCustodyDataColumnSidecarByRoot(identifier)
+        .exceptionally(
+            error -> {
+              LOG.warn(
+                  "Failed to retrieve data column sidecar for identifier {}: {}",
+                  identifier,
+                  ExceptionUtil.getMessageOrSimpleName(error));
+              return Optional.empty();
```

Comment:
the warning here is debatable, but I think returning partial result seems reasonable.
I if we fail everything we get downscored anyway?

Reply:
this warning will spam without throttling very badly, I'd remove it